shotgun_api3/shotgun.py

```diff
@@ -2219,33 +2215,16 @@ def _upload_to_storage(self, entity_type, entity_id, path, field_name, display_n
 
         # Step 1: get the upload url
 
-        upload_info = self._get_attachment_upload_info(is_thumbnail, filename)
+        multipart_upload = (os.path.getsize(path)  > 20000000)
```

Comment:
nit: on the toolkit side we would probably prefix with is to indicate boolean, e.g. `is_multipart_upload`